src/Main.elm

```diff
@@ -35,13 +37,22 @@ update msg model =
     Decrement ->
       model - 1
 
+    Reset ->
+      init
+
 
 -- VIEW
 
 view : Model -> Html Msg
 view model =
-  div []
-    [ button [ onClick Decrement ] [ text "-" ]
-    , div [] [ text (String.fromInt model) ]
-    , button [ onClick Increment ] [ text "+" ]
-    ]
+    div []
+        [ button [ onClick Decrement ] [ text "-" ]
```

Comment:
is the definition of Increment and Decrement down here, and Elm is okay with finding it later (after the definition as a type?)

Reply:
No, the definition of Increment and Decrement is on line 29. Increment and Decrement have to exist in the namespace for the view function to be able to use them.